lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-credential.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_AnsibleTower_ConfigurationManager_Credential < MiqAeServiceModelBase
```

Comment:
@jameswnl @gmcculloug 
Wouldn't this need a reference to the Authentications table?